src/Tools/ExternalAccess/Razor/Cohost/Handlers/Diagnostics.cs

```diff
@@ -21,7 +21,8 @@ internal static class Diagnostics
         var diagnosticAnalyzerService = document.Project.Solution.Services.ExportProvider.GetService<IDiagnosticAnalyzerService>();
 
         var diagnostics = await diagnosticAnalyzerService.GetDiagnosticsForSpanAsync(
-            document, range: null, DiagnosticKind.All, includeSuppressedDiagnostics: false, cancellationToken).ConfigureAwait(false);
+            document, range: null, DiagnosticKind.All, cancellationToken).ConfigureAwait(false);
+        diagnostics = diagnostics.WhereAsArray(d => !d.IsSuppressed);
```

Comment:
>WhereAsArray [](http://example.com/codeflow?start=34&length=12)

nit: could just do the check in the loop

Reply:
done.